src/screens/SettingsSheet/components/Backups/WalletsAndBackup.tsx

```diff
@@ -645,48 +623,59 @@ export const WalletsAndBackup = () => {
                 />
               </Menu>
 
-              <Menu
-                description={
-                  <Text color="secondary60 (Deprecated)" size="14px / 19px (Deprecated)" weight="regular">
-                    {i18n.t(i18n.l.wallet.back_ups.cloud_backup_description, {
-                      cloudPlatform,
-                    })}
+              <Box>
+                <Menu
+                  description={
+                    <Text color="secondary60 (Deprecated)" size="14px / 19px (Deprecated)" weight="regular">
+                      {i18n.t(i18n.l.wallet.back_ups.cloud_backup_description, {
+                        cloudPlatform,
+                      })}
 
-                    <Text onPress={onPressLearnMoreAboutCloudBackups} color="blue" size="14px / 19px (Deprecated)" weight="medium">
-                      {' '}
-                      {i18n.t(i18n.l.wallet.back_ups.cloud_backup_link)}
+                      <Text onPress={onPressLearnMoreAboutCloudBackups} color="blue" size="14px / 19px (Deprecated)" weight="medium">
+                        {' '}
+                        {i18n.t(i18n.l.wallet.back_ups.cloud_backup_link)}
+                      </Text>
                     </Text>
-                  </Text>
-                }
-              >
-                <BackUpMenuItem
-                  title={i18n.t(i18n.l.back_up.cloud.enable_cloud_backups)}
-                  loading={loading}
-                  onPress={backupAllNonBackedUpWalletsTocloud}
-                />
-              </Menu>
+                  }
+                >
+                  <BackUpMenuItem
+                    title={i18n.t(i18n.l.back_up.cloud.enable_cloud_backups)}
+                    backupState={status}
+                    onPress={backupAllNonBackedUpWalletsTocloud}
+                    disabled={status !== CloudBackupState.Ready}
+                  />
+                </Menu>
+              </Box>
             </Stack>
           </Stack>
         );
       }
     }
   }, [
     backupProvider,
-    loading,
-    backupAllNonBackedUpWalletsTocloud,
+    iconStatusType,
+    text,
+    status,
+    isCloudBackupDisabled,
+    enableCloudBackups,
     sortedWallets,
     onCreateNewSecretPhrase,
-    onViewCloudBackups,
-    manageCloudBackups,
     navigate,
     onNavigateToWalletView,
     allBackedUp,
     mostRecentBackup,
-    lastBackupDate,
+    backupAllNonBackedUpWalletsTocloud,
+    onViewCloudBackups,
+    manageCloudBackups,
     onPressLearnMoreAboutCloudBackups,
   ]);
 
-  return <MenuContainer>{renderView()}</MenuContainer>;
+  return (
+    <MenuContainer>
+      <AbsolutePortalRoot style={{ zIndex: 100 }} />
```

Comment:
ah here this file was collapsed mb
I don't understand how the navigator works really, in the network switcher pr I added this `<AbsolutePortalRoot />` to the root of the app right next to `<Routes />` and it works on most screens, but not like in the swaps thats kinda of a sheet route, in that we have another `<AbsolutePortalRoot />` specific for that route, but ideally we should have only one of this component for the whole app, or change the singleton store that controls it
doesn't having a single `<AbsolutePortalRoot />` with a big zIndex like that work? I don't think I tried that

Reply:
are you opposed to this being a Zustand store?

Reply:
nope, a zustand store would work the same

but if we can't have a single instance of this component, it has to be a context provider that creates the store and passes it to it's children
else every place we add `<AbsolutePortalRoot />` it's gonna render all nodes in the store

Reply:
I see what you mean now. Okay, I'll wait to land this until the trending tokens UI work has landed so I can refactor it.